sql/src/main/java/io/crate/planner/DecommissionNodePlan.java

```diff
@@ -78,7 +79,8 @@ public void executeOrFail(DependencyCarrier dependencies,
         );
     }
 
-    private static String resolveNodeIdFromIdOrName(DiscoveryNodes nodes, String nodeIdOrName) {
+    @VisibleForTesting
+    public static String resolveNodeIdFromIdOrName(DiscoveryNodes nodes, String nodeIdOrName) {
```

Comment:
Should be sufficient to make this package private if the test is in the same package (which should be the case for unit tests)